src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+/**
+ * Class for Ui components of Duke.
+ */
+public class Ui {
+
+    public static String divider() {

```

Comment:
Is there a reason why the divider is returned in a method instead of being stored in a constant?